README.md

```diff
@@ -25,82 +56,70 @@ The API version is specified through the accept header, as follows:
 
-The `swagger.yaml` file that is generated is defined by a combination of "global" settings in `spec/swagger_helper.rb` and indivual spec files that are, by our convention, stored in `spec/requests/swagger_docs/<version>/*.spec.rb`.
 
-You can generate a new endpoint spec file using:
-```sh
-rails generate rspec:swagger MyController
-```
+## Logic flow of the calculations
 
-You can update an existing endpoint by modifying it's spec and then running:
-```sh
-rake rswag:specs:swaggerize
-```
+The `AssessmentController` calls the `MainWorkflow`, which immediately branches off to the `PassportedWorkflow` or `UnpassportedWorkflow`.  The main difference is that unpassported applications are assessed on capital, gross income and disposable income, whereas passported applications are only assessed on capital.
+
+In each case, the workflow takes each of the assessments in turn, calls a collator to look at all the sub-objects under the summary and come up with a total figure in the case of capital, or a monthly equivalent figure in the case of gross or disposable  income, and these results are stored on the associated summary object.  After collation, an assessor is called for each summary which stores the results (eligible, ineligible, contribution required) in each of the eligibility objects (one for each proceeding type).  Finally, the main assessor is called to work out the overall result for each proceeding type taking into account the results from the capital, gross income and disposable income assessments.  The assessments controller then calls the main decorator to format the output in the required structure to send back to the client.
+
+# Development
 
 ## Setting the env vars
-To run the integration tests you will need to set up a `.env` file in the root folder.
-
-It should contain the following values:
-```shell script
-GOOGLE_SHEETS_PRIVATE_KEY_ID
-GOOGLE_SHEETS_PRIVATE_KEY
-GOOGLE_SHEETS_CLIENT_EMAIL
-GOOGLE_SHEETS_CLIENT_ID
-ALLOW_FUTURE_SUBMISSION_DATE
-LEGAL_FRAMEWORK_API_HOST
-```
 
-Set ALLOW_FUTURE_SUBMISSION_DATE to true to allow integration tests to run with submission dates that are in the future.
-A copy of the `.env` file including the current values can be found in the `Shared-LAA` section of LastPass (reach out to the team if you don't have access).
+Developers need a `.env` file in the root folder of their clone of this repo.
 
-## Threshold configuration files
+For running locally it should contain the following values:
 
-Files holding details of all thresholds values used in calculating eligibility are stored in `config/thresholds`.
-The file `values.yml` details the start dates for each set of thresholds, and the name of the file from which they should be read.
+```sh
+ALLOW_FUTURE_SUBMISSION_DATE=true
+LEGAL_FRAMEWORK_API_HOST=https://legal-framework-api-staging.apps.live-1.cloud-platform.service.justice.gov.uk
```

Comment:
These 2 are actually no longer used AFAIK  - certainly ALLOW_FUTURE_SUBMISSION_DATE appears to be long gone (and is always true)

Reply:
LEGAL_FRAMEWORK_API_HOST is still needed, I believe?

I'll ask Apply whether they would miss ALLOW_FUTURE_SUBMISSION_DATE functionality